Makefile

```diff
@@ -0,0 +1,11 @@
+BTAG=latest
+
+deb:
+	mvn package deb:package -pl web -PdebianPackage -DskipTests ${DEPLOY_OPTS}
+
+docker:
+	cd web
+	../mvnw package docker:build -Pdocker -DdockerImageName=georchestra/geonetwork -DdockerImageTags=${BTAG},latest -DskipTests -ntp
```

Comment:
`docker:build` not necessary anymore, as the build step from the docker maven plugin has been attached to the package phase, just activating the docker profile will make it:
https://github.com/georchestra/geonetwork/blob/georchestra-gn4.4.x/web/pom.xml#L1662-L1667

Reply:
maybe we could do the same with the debian profile, in order to have a harmonized approach accross the build.